animal-sniffer/src/main/java/org/codehaus/mojo/animal_sniffer/Main.java

```diff
@@ -85,7 +84,7 @@ public static void main( String[] args )
 
             try {
                 files.add(Paths.get(new URI(args[i]))); // attempt to treat it as an URI
-            } catch (URISyntaxException e) {
+            } catch (Exception e) {
```

Comment:
Might be good to add a comment why catching `URISyntaxException` is not enough. Otherwise someone might try refactoring this in the future and by accident breaks the parsing.